server/src/models/constants.ts

```diff
@@ -51,3 +51,19 @@ export enum Status {
     Deaths = 'deaths',
     Recovered = 'recovered',
 }
+
+export enum LogLevel {
+//   Emerg = 'emerg',
```

Comment:
what are these?

Reply:
These are log levels supported by winston. But when I try to log messages to loggly. Just those uncommented works. I'm still in process of figuring out how to fix it